src/howitz/endpoints.py

```diff
@@ -37,6 +38,40 @@ class EventColor(StrEnum):
     DEFAULT = ""
 
 
+# Inspired by https://stackoverflow.com/a/54732120
+class EventSorting(Enum):
```

Comment:
Not quite happy with the name of this enum. `EventSort`? `EventSortKey`? `EventSortConfig`?